app/models/scoreboard.rb

```diff
@@ -24,14 +24,17 @@ def colspec_is_well_formed
 
     # The parse will throw an exception if the string has a JSON syntax error
     begin
-      # Quote JSON keys and values if they are not already quoted
-      quoted = colspec.gsub(/([a-zA-Z0-9]+):/, '"\1":').gsub(/:([a-zA-Z0-9]+)/, ':"\1"')
```

Comment:
I remove the quoting here because otherwise unquoted colspecs will pass validations. This causes problems in `show` and `scoreboardOrderSubmissions` which do not perform quoting. Rather than quote at all three places, it makes more sense to enforce good syntax from the user.